metabase/metabase
# Fix webpack memory leaks in watch mode

### Description

Reduce noise from webpack, attempt to make re-calculations a bit lighter without using hashes in dev mode

### How to verify

- run `yarn build-hot` and check the output
- edit some file, check output again

### Demo

https://www.loom.com/share/ca39d705695943ab88f0084603e50f69

compare memory usage
https://www.loom.com/share/cda3d0350b334f9b8bed5aeda81ca803
